internal/command/command.go

```diff
@@ -114,7 +158,19 @@ func (c Client) SearchForFlags(flags []string, ctxLines int, delimiters []rune)
 		return nil, errors.New(string(out))
 	}
 
-	grepRegexWithFilteredPath, err := regexp.Compile("(?:" + regexp.QuoteMeta(c.Workspace) + "/)" + grepRegex.String())
+	/*
+	   searchRegex splits resulting grep lines into groups
+	   Group 1: File path.
+	   Group 2: Line number.
+	   Group 3: Separator. A colon indicates a match, a hyphen indicates a context lines
+	   Group 4: Line contents.
+
+	   Example output from line search tool: /path/to/file/flags.go:2:ldClient.BoolVariation("my-flag-key", user, false)
+	   Example result: ["/path/to/file/flags.go:2:ldClient.BoolVariation(\"my-flag-key\", user, false)", "/path/to/file/flags.go", 2, ":", "ldClient.BoolVariation(\"my-flag-key\", user, false)"]
+	*/
+	searchRegex := regexp.MustCompile("(.*)[:-]([0-9]+)(:|-)(.*)")
```

Comment:
This regex moved, so the diff won't show that it did change a bit:
`([^:]+)(:|-)([0-9]+)[:-](.*)` -> `(.*)[:-]([0-9]+)(:|-)(.*)`

This change matches the content slightly differently, for a couple reasons:
- `rg` "context" lines look slight different from the ones returned from `ag` in that the separator between the filepath and line number is a hyphen instead of a colon. The matching needed to be updated to match correctly while not breaking matching on filepaths with hyphens on them
- The group matching the separator was matching the one that was always a colon (bug), when we really needed to match the one that could be a colon or a hyphen. So, the regex has been updated to match the correct item. This resulted in the groups being reordered.